src/screens/HomeScreen.tsx

```diff
@@ -1,12 +1,41 @@
-/* eslint-disable react-native/no-inline-styles */
-import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import React, { useContext } from 'react';
+import {
+  View, Text, Button, StyleSheet, StatusBar, ImageBackground,
+} from 'react-native';
+
+import { ThemeContext } from '../../App.tsx';
+import darkImage from '../../assets/launchScreen.png';
+import lightImage from '../../assets/sp.png';
+import COLORS from '../colors/myColors.tsx';
 
 function HomeScreen() {
+  type ThemeContextType = {
+    theme: string;
+    // eslint-disable-next-line no-unused-vars
+    setTheme: (theme: string) => void;
+  };
+
+  const { theme, setTheme } = useContext(ThemeContext) as ThemeContextType;
   return (
-    <View style={styles.containerStyle}>
-      <Text style={styles.textStyles}>HomeScreen</Text>
-    </View>
+    <ImageBackground
+      defaultSource={theme === 'dark' ? darkImage : lightImage}
+      source={theme === 'dark' ? darkImage : lightImage}
+      style={styles.imgStyles}
+      resizeMode="cover"
+    >
+      <View style={styles.containerStyle}>
+        <StatusBar barStyle={theme === 'dark' ? 'light-content' : 'dark-content'} />
```

Comment:
@muhammadhassan2k3 will we use these conditionals through out the application ?

Reply:
@muhammadhassan2k3 we should first standardise all the colors names in the app. for example, for all the headings in the Header of the screens, we can name it `screenTitle` and then define it something like following:

```typescript

export const THEME_COLORS = {
   light: {
     screenTitle: "#fff",
     leftHeaderIcon: "#fff",
   },
   dark: {
     screenTitle: "#000"
   }
}

```

then we can write a utility function like following to get the color based upon theme & color name

```typescript
export const getColorByName (theme: Theme, colorName: ColorNames) {
   return THEME_COLORS?.[theme]?.[colorName];
}
```


OR we can create hooks like following:

```
useColors () {
   const theme = useTheme();
   return THEME_COLORS[theme];
}
```

then wherever, we want to use colors we use it like:

```
const colors = useColors()
const screenTitle = colors?.screenTitle;
```